spec/models/instrument_spec.rb

```diff
@@ -488,6 +488,15 @@
       assert_equal 45, @next_reservation.reserve_start_at.min
     end
 
+    it "should find next available reservation with cutoff hours" do
+      expect(@rule.instrument.update_attribute :cutoff_hours, 10).to be true
```

Comment:
If you're going to do more edits, go with `it "finds the next available…"` instead of `it "should find…"`